tests/Avalonia.Visuals.UnitTests/Media/TextFormatting/BiDiPairedBracketTypeTests.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Avalonia.Visuals.UnitTests.Media.TextFormatting
+{
+    public class BiDiPairedBracketTypeTests
```

Comment:
Nit: Are you planning to add tests for that or should we just delete it for now?

Reply:
This can be removed